File: JAVA/164. Maximum Gap.java
class Solution {
    public int maxConsecutive(int b, int t, int[] s) {
        Arrays.sort(s);
        int m=Math.max(t-s[s.length-1],s[0]-b);
        for(int i=0;i<s.length-1;i++){
            m=Math.max(m,s[i+1]-s[i]-1);
        }
        return m;
    }
}

File: C++/164. Maximum Gap.cpp
class Solution {
public:
    int maximumGap(vector<int>& n) {
        sort(n.begin(),n.end());
        int m=0;
        for(int i=0;i<n.size()-1;i++)
        m=max(m,n[i+1]-n[i]);
        return m;
    }
};